waiter/integration/waiter/token_request_test.clj

```diff
@@ -474,7 +475,8 @@
           (let [token-response (get-token waiter-url token)
                 response-body (-> token-response (:body) (json/read-str) (pc/keywordize-map))]
             (is (nil? (get response-body :run-as-user)))
-            (is (= (assoc service-description :owner (retrieve-username)) response-body))))
+            (is (= (assoc service-description :token-metadata {:owner (retrieve-username)})
```

Comment:
This will break the UI and any other scripts that are expecting `owner` to be in the main map.